FinniversKit/Sources/Components/TextField/TextField.swift

```diff
@@ -137,6 +137,7 @@ public class TextField: UIView {
 
     public let inputType: InputType
     public var phoneNumberRegEx = "^(?:\\s*\\d){8,11}$"
+    public var emailRegEx = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,64}"
```

Comment:
Do we need to change this anywhere in the app, or can it be `let` instead?

Reply:
I think the first person who implemented this just uses vars